src/interactive-guidance/index.njk

```diff
@@ -0,0 +1,94 @@
+  <li>get an estimate of financial impact (owed/refunded) ahead of taking an action</li>
+  <li>get a (non-legally-binding) statement of eligibility or status</li>
+</ul>
+
+<h2 class="govuk-heading-l" id="when-not-to-use">
+  When not to use
+</h2>
+
+<p class="govuk-body">
+Do not use this pattern:
+</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>to publish generic information that users need and expect to find quickly and easily, like tax rates or simple eligibility requirements - they should be able to find this on a single page of GOV.UK guidance directly from search engines (interactive guidance questions and outcomes aren’t indexed by external or internal search)</li>
+  <li>to provide information that they will then need to re-enter in a transactional service after signing in (other than the bare minimum required to work out whether or not they need to use a service, where this is too complex to cover in GOV.UK guidance)</li>
+  <li>to explain a policy that’s so complex, variable or dependent on circumstances that users cannot be realistically expected to self-serve (and where this isn’t clear, it should be validated by research)</li>
+</ul>
+
+<h2 class="govuk-heading-l" id="examples">
+  Examples
+</h2>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li><a class="govuk-link" href="https://tax.service.gov.uk/guidance/check-non-paye-income">Check if you need to pay tax on additional income</a></li>
+  <li><a class="govuk-link" href="https://www.gov.uk/check-if-you-need-tax-return">Check if you need to do a tax return</a></li>
+  <li>Estimate Inheritance Tax</li>
+</ul>
+
+<h2 class="govuk-heading-l" id="how-it-works">
+  How it works
+</h2>
+
+<p class="govuk-body">
+The HMRC interactive guidance team designs interactive guidance products. They also work with relevant service or GOV.UK teams on the wider user journeys from related guidance content and on into transactional services.
+</p>
+
+<p class="govuk-body">
+Contact the HMRC GOV.UK team or your design lead if you have a proposal for interactive guidance.
+</p>
+
+<p class="govuk-body">
+Interactive guidance must be linked to from GOV.UK. The format of this depends on whether the underlying user need is:
+</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>‘mainstream’ for the general public or small businesses (managed by the GDS GOV.UK team)</li>
+  <li>‘specialist’ or expert users with some prior knowledge (managed by the HMRC GOV.UK team)</li>
+</ul>
+
+<p class="govuk-body">
+Interactive guidance should complement existing GOV.UK content rather than replacing it. This is to make sure that users can easily revisit or find information they need, because the information on outcome pages is not indexed by search engines or accessible via GOV.UK navigation. Also, more experienced or specialist users (such as agents) often need to see all of the rules in one place.
```

Comment:
Should this be 'replace' rather than 'replacing'?

Reply:
I'm not a content designer but I think a way to write this with the gerund would be 'Interactive guidance should be complementing existing GOV.UK content rather than replacing it'?